benefit/data/sheets/FixedLists/FixedLists.json

```diff
@@ -760,160 +760,141 @@
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "208506", "ListElement": "Enfield"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "815811", "ListElement": "Exeter"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "406867", "ListElement": "Fox Court (S)"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "427519", "ListElement": "Gateshead"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "366559", "ListElement": "Glasgow"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "198592", "ListElement": "Gloucester"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "641199", "ListElement": "Grimsby"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "723075", "ListElement": "Hamilton"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "784691", "ListElement": "Hastings"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "386417", "ListElement": "Hatton Cross"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "741758", "ListElement": "Havant"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "700596", "ListElement": "Haverfordwest"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "228883", "ListElement": "Hereford JC"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "437303", "ListElement": "High Wycombe"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "197852", "ListElement": "Huddersfield"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "362420", "ListElement": "Hull Mags"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "900001", "ListElement": "Inverness"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "287515", "ListElement": "Ipswich"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "416742", "ListElement": "Newport"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "767032", "ListElement": "Kidderminster"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "671879", "ListElement": "Kings Lynn Mags"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "353615", "ListElement": "Lancaster"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "495952", "ListElement": "Leeds"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "500064", "ListElement": "Leicester"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "195465", "ListElement": "Lincoln"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "345663", "ListElement": "Liverpool 2"},
```

Comment:
is Liverpool2 valid?

Reply:
![image](https://user-images.githubusercontent.com/92032149/187410084-8fe57480-2279-48d0-ae2f-3cddd1d0f706.png)
Yes

Reply:
What file are you using?

Reply:
I don't think we want to be adding inactive venues @pk2453

Reply:
This is what provided in the Jira. Column U from this sheet. @dattipoeHMCTS